pytest_html/plugin.py

```diff
@@ -157,24 +158,24 @@ def create_asset(self, content, extra_index,
         def append_extra_html(self, extra, extra_index, test_index):
             href = None
             if extra.get('format') == extras.FORMAT_IMAGE:
-                if self.self_contained:
+                content = extra.get('content')
+                if re.match('^file|^http', content):
+                    html_div = html.img(src=content)
+                elif self.self_contained:
```

Comment:
hmm... so this means if the `--self-contained-html` flag is passed, these images will not be displayed in the report at all. Whilst this is possibly acceptable (I don't think we should be downloading/reading these files) it could be confusing to users. Perhaps we could add a warning to the documentation that standalone reports may still include assets referenced by URL? We could also log a warning for each URL based asset when we're in standalone mode.